apps/bfd-server/bfd-server-launcher/src/test/java/gov/cms/bfd/server/launcher/DataServerLauncherAppIT.java

```diff
@@ -56,6 +56,7 @@ public final class DataServerLauncherAppIT {
           .add("([0-9]+|-)") // BlueButton-UserId
           .add("\"([^\"]*)\"") // BlueButton-User
           .add("(\\S+)") // BlueButton-BeneficiaryId
+          .add("(\\S+)") // Response Header X-Request-ID
```

Comment:
Just noting for anyone else who's wondering (since I was): this **is** used (down around line 200) to verify that the local access log's lines actually match this pattern.